cmd_configure.go

```diff
@@ -38,14 +38,24 @@ func ConfigureDefaultCommand(ctx ConfigureCommandContext) error {
 }
 
 func ConfigureGithubCommand(ctx ConfigureCommandContext) error {
+	cfg, err := config.Open()
+	if err != nil {
+		return NewExitError(fmt.Errorf("Failed to open config file: %v", err), 1)
+	}
+
 	ctx.Header("Let's configure your github.com credentials 💻")
 
-	ctx.Printf("We need to authorize this app to access your repositories. " +
-		"This authorization is stored locally, buildkite.com never gets access to it.\n\n")
+	ctx.Printf(`We need to authorize this app to access your repositories.
+This authorization is stored locally in ` + cfg.Path + `, buildkite.com never gets access to it.
+
+`)
```

Comment:
i really wish go had something like ruby's [squiggly heredoc](https://infinum.com/blog/multiline-strings-ruby-2-3-0-the-squiggly-heredoc/), but alas, the maintainers would never go for it.